applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, model, params ):
                 "model_part_name"           : "",
                 "interval"                  : [0.0, 1e30],
                 "write_drag_output_file"    : true,
+                "output_folder"             : "TimeBasedAsciiResults",
```

Comment:
why not "DragResults"?

Reply:
I would by default put it into TimeBasedAsciiResults, which should serve as a collector. The user shoud by able to specify other directories and subfolders if they wanted to. Otherwise, instead of getting a bunch of *.dat files in the current directory, will get almost the similar amount of output folders, which as far as I am concerned is similarly bad.

Reply:
Now it works -> "Btw does it also work for e.g. "output_folder_name" = "multiple/paths/before/actual/folder"?"